### PR TITLE
docs: updates docs to be clearer on running example

### LINKED ISSUE #16: components folder missing from examples
Found a bug? Maybe our [Slack Community](https://slack.cloudposse.com) can help. 

[Slack Community](https://slack.cloudposse.com)

## Describe the Bug
Getting started guide mentions a components folder in the example file. That folder does not exist. Running `atmos terraform apply eks -s ue2-dev` returns the following error

	* step "terraform workspace": job "terraform workspace": job "terraform shell": job "shell": command "bash -c /usr/bin/terraform-0.13 workspace select ue2-dev || /usr/bin/terraform-0.13 workspace new ue2-dev" in "./components/terraform/eks": chdir ./components/terraform/eks: no such file or directory

## Steps to Reproduce
1. git clone git@github.com:cloudposse/atmos.git
2. cd atmos
3. cd example
4. make all
5. assume-role
6. atmos terraform plan eks --stack=ue2-dev
7.  atmos terraform apply eks --stack=ue2-dev

## Screenshots
If applicable, add screenshots or logs to help explain your problem.

## Environment:

Anything that will help us triage the bug will help. Here are some ideas:
 - OS: [e.g. Linux, OSX, WSL, etc]
 - Version [e.g. 10.15]

### PR DESCRIPTION
**Pull request by Gowiem**
## what
* Updates the example docs a bit to mention additional steps + call out our tutorial on atmos  

## why
* It should no longer be confusing that you need to run `vendir sync`

## references
* Fixes #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,7 +167,9 @@ Our recommended filesystem layout looks like this:
 
 ## Example
 
-The [example](example) folder contains a complete solution that shows how to:
+**NOTE**: Our recommended example of atmos is found on our documentation site: https://docs.cloudposse.com/tutorials/atmos-getting-started/
```
Comment (Gowiem) [inline README.md:170]: @osterman @aknysh I'm wondering if we should strike this example altogether and just direct folks to the docs site / tutorial... I feel like maintaining both is a bit cumbersome and the tutorial does a decent walkthrough. What do you two think?

Comment (aknysh) [reply README.md:170]: @Gowiem sounds good. We can maintain the doc in one place

Comment (mikedizon) [reply README.md:170]: The tutorial doesn't cover everything in the example. Setting up/using `vendir` is missing.  Can the docs be updated before the example gets removed?

**Comment by aknysh** [comment]
closing as not relevant to new `atmos` written in `Go`

**Pull request closed without merging**